modules/standard/Random.chpl

```diff
@@ -1041,7 +1046,10 @@ module Random {
         :arg arr: The array to be filled
         :type arr: [] :type:`eltType`
       */
-      proc fillRandom(arr: [] eltType) {
+      proc fillRandom(arr: [] eltType) throws {
```

Comment:
Does it throw?

Reply:
sorry, I thought compileErrors are thrown. Will correct this